src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,26 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
+        Machine excavation = new Excavator();
+        System.out.println(bulldozer.doWork());
+        System.out.println(truck.doWork());
+        System.out.println(excavation.doWork());
+        /*
+        System.out.println(bulldozer.stopWork());
+        System.out.println(truck.stopWork());
+        System.out.println(excavation.stopWork());
+        */
+        Machine[] machine1 = new Machine[]{bulldozer, truck, excavation};
+        for (Machine machine : machine1) {
+            println(machine);
+        }
+    }
+
+    public static void println(Machine machine) {
+        System.out.println(machine.stopWork());
+    }
```

Comment:
here you need to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop